src/05_lists.py

```diff
@@ -8,22 +8,26 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE 
-print(x)
+print(x.append(4))
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE 
-print(x)
+print(x.extend(y))
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE 
-print(x)
+print(x.pop(4 & 5))
```

Comment:
I've never seen `pop` with multiple indexes passed to it before like `(4 & 5)`, but it seems to work! Alternatives that may be more readable would be `x.remove(8)` to search and remove the passed number(8) or `x.pop(4)`

Reply:
I will take a look at the other ways to do it